types/prosemirror-markdown/index.d.ts

```diff
@@ -119,7 +120,9 @@ export class MarkdownParser<S extends Schema = any> {
  * A parser parsing unextended [CommonMark](http://commonmark.org/),
  * without inline HTML, and producing a document in the basic schema.
  */
-export let defaultMarkdownParser: MarkdownParser;
+export const defaultMarkdownParser: MarkdownParser;
+
+export const schema: Schema;
```

Comment:
Please add the [comment from the source code](https://github.com/ProseMirror/prosemirror-markdown/blob/65caf3c67c0946f61aa7898d16ad027d898905fc/src/schema.js#L3):

```typescript
/**
 * Document schema for the data model used by CommonMark.
 */
```